src/Servers/Kestrel/test/InMemory.FunctionalTests/Http3/Http3ConnectionTests.cs

```diff
@@ -73,6 +74,58 @@ public async Task CreateRequestStream_RequestCompleted_Disposed()
             Assert.True(requestStream.Disposed);
         }
 
+        [Fact]
+        public async Task HEADERS_Received_ContainsExpect100Continue_100ContinueSent()
```

Comment:
Note to self: rename this. This name was taken from the h2 equivalent of this test but doesn't conform with the convention used here.

Reply:
Actually, I changed my mind. Will leave it as-is so we can easily correlate the two versions of this test. FWIW there are other tests that use this pattern in this file too.